includes/class-wc-query.php

```diff
@@ -533,14 +533,76 @@ public function get_meta_query( $meta_query = array(), $main_query = false ) {
 		if ( ! is_array( $meta_query ) ) {
 			$meta_query = array();
 		}
-
 		$meta_query['price_filter']  = $this->price_filter_meta_query();
-		$meta_query['rating_filter'] = $this->rating_filter_meta_query();
-
 		return array_filter( apply_filters( 'woocommerce_product_query_meta_query', $meta_query, $this ) );
 	}
 
 	/**
+	 * Appends tax queries to an array.
+	 * @param array $tax_query
+	 * @param bool  $main_query
+	 * @return array
+	 */
+	public function get_tax_query( $tax_query = array(), $main_query = false ) {
+		if ( ! is_array( $tax_query ) ) {
+			$tax_query = array( 'relation' => 'AND' );
+		}
+
+		// Layered nav filters on terms.
+		if ( $main_query && ( $_chosen_attributes = $this->get_layered_nav_chosen_attributes() ) ) {
+			foreach ( $_chosen_attributes as $taxonomy => $data ) {
+				$tax_query[] = array(
+					'taxonomy'         => $taxonomy,
+					'field'            => 'slug',
+					'terms'            => $data['terms'],
+					'operator'         => 'and' === $data['query_type'] ? 'AND' : 'IN',
+					'include_children' => false,
+				);
+			}
+		}
+
+		$product_visibiity_terms   = wc_get_product_visibility_term_ids();
```

Comment:
You have a typo in your variable name. You have `visibiity` here instead of `visibility`.
It looks like the same typo is used in the usage of it too -- so make sure to update all uses.